### DISCUSSION TITLE
Easier way to get Device Key

### DISCUSSION THREAD

**Discussion by tamueller** [vinodsr/node-red-contrib-tuya-smart-device#103] [Ideas]
Hi,
I just found this method of getting your device key, which does not include the CLI, and seems a bit easier.
https://github-wiki-see.page/m/iRayanKhan/homebridge-tuya/wiki/Get-Local-Keys-for-your-devices

Maybe you want to link this as an alternative way to get the device key.

**Comment by vinodsr** [comment]
Looks interesting.

Adding Task

**Comment by vinodsr** [comment]
This has been added
